sanitize.go

```diff
@@ -5,6 +5,8 @@ package dns
 // rrs.
 // m is used to store the RRs temporary. If it is nil a new map will be allocated.
 func Dedup(rrs []RR, m map[string]RR) []RR {
+	println("dns: Dedup: this function is deprecated")
```

Comment:
The Deprecated doc comment should be added to this too: https://rakyll.org/deprecated/